package.json

```diff
@@ -62,7 +62,8 @@
     "lodash": "^3.10.1",
     "minimist": "*",
     "moment-timezone": "^0.5.11",
-    "node-schedule": "^0.1.13"
+    "node-schedule": "^0.1.13",
+    "winston-color": "^1.0.0"
```

Comment:
I think it'd be a better semantic description of our dependencies to include `winston` explicitly, instead of relying on `winston-color` to pull it in. Then we could also pin/control `winston` versions more easily.